bin/config.mk.defaults.osx

```diff
@@ -59,7 +59,7 @@ export VKDT_USE_ALSA
 
 # build the i-mcraw loader module to load raw video from the motioncam app.
 # it has no additional dependencies but comes with several ten thousand lines of c++ code
-VKDT_USE_MCRAW=1
+VKDT_USE_MCRAW=0
```

Comment:
does that not work? newer master should have removed the SSE/x86 code in this module. i don't see a reason why it wouldn't work on arm now.